src/lexer/common.ts

```diff
@@ -72,21 +88,27 @@ export function consumeMultiUnitCodePoint(parser: ParserState, hi: number): 0 |
 /**
  * Use to consume a line feed instead of `consumeLineBreak`.
  */
-export function consumeLineFeed(parser: ParserState, state: LexerState): void {
-  advanceAndLawrenceDolTheCRLF(parser);
-  parser.flags |= Flags.NewLine;
-  if ((state & LexerState.LastIsCR) === 0) {
-    parser.column = 0;
-    parser.line++;
-  }
-}
-
-export function consumeLineBreak(parser: ParserState): void {
-  parser.flags |= Flags.NewLine;
+export function consumeLineBreak(parser) {
+  parser.flags |= 1;
```

Comment:
Avoid using number here. Use the const enum value for readability. The compiler will fix the number part

Reply:
I started off working on the generated JS, so the constants had been eliminated. This was an oversight.